### ISSUE TITLE
`require-specific-component` does not raise an error when only one prop is passed in the `<Box>` component

### ISSUE THREAD

**Issue by thiagohflima** [yukukotani/eslint-plugin-chakra-ui#143]
## Problem
Currently, the `require-specific-component` feature does not raise an error when only one prop is passed to the `<Box>` component. For example:

```jsx
<Box display='flex'>
  test1
</Box>
```

## Expected Behavior
The `require-specific-component` should raise an error in this scenario, indicating that multiple props should be provided.
However, when a second prop is added (such as padding, margin, color, etc.), the `require-specific-component` feature works as expected. For instance:

```jsx
<Box display='flex' p={2}>
  test2
</Box>
```

This results in:

```jsx
<Flex p={2}>
  test2
</Flex>
```

## Request
I kindly request clarification on whether the current behavior is intentional or not. Thank you for your attention to this matter.

---
### Edit
Instead of referencing `props-order`, I should have mentioned `require-specific-component`. I just fixed it.

**Comment by yukukotani** [comment]
@thiagohflima Thanks for the reporting!

Could you provide an example of expected behavior and current behavior? I found the bug of `require-specific-component` below but is this what you are saying?

### Input

```tsx
<Box display="flex">
  body
</Box>
```

### Expected

```tsx
<Flex>
  body
</Flex>
```

### Current

No error

**Comment by thiagohflima** [comment]
@yukukotani, the example you provided is exactly what I observed.

I would expect that any `<Box>` component with the prop `display="flex"` would be transformed into a `<Flex>` component, but it seems that this transformation only occurs if the `<Box>` component has at least two props.

**Comment by yukukotani** [comment]
@thiagohflima Got it, thanks!

Could you try to fix this? I think the problem is around here:

https://github.com/yukukotani/eslint-plugin-chakra-ui/blob/0818794e3f23dfd0d69d3c437fee619abab58255/src/rules/require-specific-component.ts#L42-L53

**Comment by thiagohflima** [comment]
@yukukotani I'll take a look and try to open a PR to fix that as soon as possible. Thanks for your attention!

**Comment by yukukotani** [comment]
closed by #144
